proto/v1/grafeas.proto

```diff
@@ -444,6 +444,12 @@ message CreateNoteRequest {
     (google.api.resource_reference).type = "grafeas.io/Project"
```

Comment:
Put these documentation changes in a separate PR.

We should keep each PR as small as possible.

Reply:
first batch would be the minimal validation lib